AstroCal/Calendar.py

```diff
@@ -0,0 +1,85 @@
+from kivy.app import App
+from kivy.uix.button import Button
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.gridlayout import GridLayout
```

Comment:
Make sure to merge the latest version of dev into your branch. When I pulled down this branch I noticed it is a few pull merges behind the dev branch.

Reply:
Have merge latest changes from dev into branch